pyproject.toml

```diff
@@ -51,6 +51,8 @@ python_files = ["test*.py"]
 
 [tool.ruff]
 line-length = 120
+
+[tool.ruff.lint]
```

Comment:
does this change something about how linting works?

Reply:
it works the same, the previous way is just deprecated: 
> warning: The top-level linter settings are deprecated in favour of their counterparts in the `lint` section. Please update the following options in `pyproject.toml`:
>  - 'select' -> 'lint.select'

This can also be changed this to `lint.select` under `[tool.ruff]` to prevent adding another section and get the same effect

Reply:
Cool, thanks for updating!